frontend/simple/js/state.js

```diff
@@ -19,6 +21,8 @@ var store // this is set and made the default export at the bottom of the file.
 // NOTE: THE STATE CAN ONLY STORE *SERIALIZABLE* OBJECTS! THAT MEANS IF YOU TRY
 //       TO STORE AN INSTANCE OF A CLASS (LIKE A CONTRACT), IT WILL NOT STORE
 //       THE ACTUAL CONTRACT, BUT JSON.STRINGIFY(CONTRACT) INSTEAD!
+
+Vue.events = new Vue() // global event bus, use: https://vuejs.org/v2/api/#Instance-Methods-Events
```

Comment:
*(This is the closest GitHub lets me comment about line 29)*

Regarding:

> contracts: {}, // contractIds => type (for contracts we've successfully subscribed to)

This contract has been broken. It is now a map of "contractId => { type:string, recentHash:string }".

If we were fully utilizing FlowType it would have given you a type error and prevented you from submitting the PR, but we're not, so please update the comment accordingly.